osu.Game.Tests/Visual/Navigation/TestScenePresentBeatmap.cs

```diff
@@ -139,7 +140,7 @@ private void presentAndConfirm(Func<BeatmapSetInfo> getImport)
             AddStep("present beatmap", () => Game.PresentBeatmap(getImport()));
 
             AddUntilStep("wait for song select", () => Game.ScreenStack.CurrentScreen is Screens.Select.SongSelect);
-            AddUntilStep("correct beatmap displayed", () => Game.Beatmap.Value.BeatmapSetInfo.ID == getImport().ID);
+            AddUntilStep("correct beatmap displayed", () => Game.Beatmap.Value.BeatmapSetInfo.MatchesOnlineID(getImport()));
```

Comment:
Technically a change in behaviour, but I think it's fine as long as tests pass.

Reply:
I think the newly-returned `.Equals()` implementation could be applied to these and work correctly. But I'm not fussed either way.

Reply:
I intentionally changed this one. Felt safer.